src/components/thumb-carousel/thumb-carousel.jsx

```diff
@@ -28,17 +29,23 @@ export default React.createClass({
     let triggers = this.props.contents.map((item, index) => {
       let classNames = `hero${this.state.activeTrigger === index ? ` active` : ``}`;
 
+      // Using anchors instead of buttons because Firefox adds "padding" to button elements even with padding:0
       return (
-        <button key={index} className={classNames} onClick={this.onTriggerClick}>
+        <a key={index} className={classNames} onClick={this.onTriggerClick}>
```

Comment:
this feels like not enough resets... https://jsfiddle.net/nfzjjdjf looks the same for me.